nautobot/users/views.py

```diff
@@ -266,6 +284,25 @@ def dispatch(self, request, *args, **kwargs):
             return self.handle_no_permission()
         return super().dispatch(request, *args, **kwargs)
 
+    def extra_message_context(self, obj):
+        """
+        Context variables for this extra message.
+        """
+        return {"new_global_default_view": obj}
+
+    def extra_message(self, **kwargs):
+        new_global_default_view = kwargs.get("new_global_default_view")
+        view_name = new_global_default_view.view
+        message = ""
+        if new_global_default_view.is_global_default:
+            message += f"<br>The global default saved View for '{view_name}' is set to <a href='{new_global_default_view.get_absolute_url()}'>{new_global_default_view.name}</a>."
+        return message
```

Comment:
Is this redundant with lines 466-474 below? If not, why not?

Reply:
This extra message function is for setting `is_global_default` successfully from the SavedView management view.
lines 466-474 were originally for setting `is_global_default` from the create saved view modal. Since we lost the functionality, lines 466-474 are no longer necessary.